### PR TITLE
Remove redundancy codes and minor correction

### PR DESCRIPTION
**Pull request by zarda**
Minor correction of the codes for measuring performance

### CODE REVIEW

File: Region/RegionHandler.cc [hunk 1]
```diff
@@ -881,7 +881,8 @@ bool RegionHandler::GetRegionHistogramData(
         auto t_end_region_histogram = std::chrono::high_resolution_clock::now();
         auto dt_region_histogram =
             std::chrono::duration_cast<std::chrono::microseconds>(t_end_region_histogram - t_start_region_histogram).count();
```
Comment (pford) [inline Region/RegionHandler.cc:883]: Out of curiosity: why do you get the count in microseconds then convert ( * 1e-3) to milliseconds?  You can get the count in milliseconds: `std::chrono::duration_cast<std::chrono::milliseconds>(t_end - t_start).count();`

**Review by pford — APPROVED** [review]
Thank you for fixing this mess.  I hope it was a merge issue and I did not repeat code this badly!

File: Region/RegionHandler.cc (hunk 1 above)
Comment (veggiesaurus) [reply Region/RegionHandler.cc:883]: > Out of curiosity: why do you get the count in microseconds then convert ( * 1e-3) to milliseconds? You can get the count in milliseconds: `std::chrono::duration_cast<std::chrono::milliseconds>(t_end - t_start).count();`

if you get it in milliseconds, doesn't it round it to the nearest millisecond?

Comment (pford) [reply Region/RegionHandler.cc:883]: Yes, you are right.

File: Session.cc [hunk 2]
```diff
@@ -947,12 +947,6 @@ void Session::OnResumeSession(const CARTA::ResumeSession& message, uint32_t requ
...
     // Measure duration for resume
-    if (_verbose_logging) {
-        auto t_end_resume = std::chrono::high_resolution_clock::now();
-        auto dt_resume = std::chrono::duration_cast<std::chrono::microseconds>(t_end_resume - t_start_resume).count();
-        fmt::print("Resume in {} ms\n", dt_resume * 1e-3);
-    }
-
     if (_verbose_logging) {
         auto t_end_resume = std::chrono::high_resolution_clock::now();
         auto dt_resume = std::chrono::duration_cast<std::chrono::milliseconds>(t_end_resume - t_start_resume).count();
```
Comment (pford) [inline Session.cc]: Change to microseconds and adjust in fmt::print

**Review by pford — CHANGES_REQUESTED** [review]

File: Session.cc (hunk 2 above)
Comment (pford) [reply Session.cc]: There were quite a few of these so I fixed them myself.

**Review by pford — APPROVED** [review]

**Comment by veggiesaurus** [comment]
closing as this PR's changes were included in #609

**Pull request closed without merging**